src/scss/forms/_radios.scss

```diff
@@ -0,0 +1,49 @@
+@use '~accoutrement/sass/tools';
+@use '../config';
+
+/// # Radio Buttons
+/// @group input-radio
+
+/// @group input-radio
```

Comment:
duplicate lines

Reply:
These are actually two different blocks. The first is adding an h1 to the input-radio page, and the second is attached to the code block below. They could maybe use more detail, but they aren't duplicate.

Reply:
I sorta wondered if it was there for just the heading... ok, makes sense.